kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,106 @@ This sample demonstrates how to install the Traefik ingress controller to provid
 ```
-$ helm install --name traefik-operator --namespace traefik --values values.yaml stable/traefik
-```
-With the dashboard enabled, you can access the Traefik dashboard with the URL `http://${HOSTNAME}:30305`, with the HTTP host `traefik.example.com`.
-```
-$ curl -H 'host: traefik.example.com' http://${HOSTNAME}:30305/
-```
-
-## Optionally, download the Traefik Helm chart
-If you want, you can download the Traefik Helm chart and untar it into a local folder:
-```
-$ helm fetch  stable/traefik --untar
-```
-
-## Update the Traefik operator
-After the Traefik operator is installed and running, if you want to change some configurations of the operator, use `helm upgrade` to achieve this.
-```
-$ helm upgrade traefik-operator stable/traefik --values values.yaml 
+$ helm install traefik-operator traefik/traefik --namespace traefik --values values.yaml
 ```
 
-## Configure Traefik as a load balancer for WLS domains
-In this section we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
+## Configure Traefik as a load balancer for WebLogic domains
+This section describes how to use Traefik to handle traffic to backend WebLogic domains.
 
-### 1. Install WLS domains
-Now we need to prepare some domains for Traefik load balancing.
+### 1. Install WebLogic domains
+First, we need to prepare two domains for Traefik load balancing.
 
-Create two WLS domains:
-- One domain with name `domain1` under namespace `default`.
-- One domain with name `domain2` under namespace `test1`.
+Create two WebLogic domains:
+- One domain with `domain1` as the domain UID and namespace `weblogic-domain1`.
+- One domain with `domain2` as the domain UID and namespace `weblogic-domain2`.
 - Each domain has a web application installed with the URL context `testwebapp`.
+- Each domain has a WebLogic cluster `cluster-1` where each Managed Server listens on port `8001`.
 
-### 2. Install the Traefik Ingress
-#### Install a host-routing Ingress
+### 2. Web request routing 
+The following sections describe how to route an application web request to the WebLogic domain through a Traefik frontend.
+
+#### Host-based routing 
+This sample demonstrates how to access an application on two WebLogic domains using host-based routing. Install a host-based routing Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute).
 ```
 $ kubectl create -f samples/host-routing.yaml
+ingressroute.traefik.containo.us/traefik-hostrouting-1 created
+ingressroute.traefik.containo.us/traefik-hostrouting-2 created
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different hostnames.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different host names as defined in route section of `host-routing.yaml` file.
```

Comment:
as defined in route section -> as defined in *the* route section (add missing word, the)
of `host-routing.yaml` file. -> of *the* `host-routing.yaml` file. (add missing word, the)

Reply:
done in latest commit